styles/fonts.css

```
@@ -0,0 +1,45 @@
+
+@font-face {
+  font-family: "Montserrat";
+  font-style: normal;
+  font-weight: 300;
+  font-display: swap;
+  src: url("/fonts/montserrat-v15-latin-300.eot");
+  src: local(""),
+    url("/fonts/montserrat-v15-latin-300.eot?#iefix")
+      format("embedded-opentype"),
+    url("/fonts/montserrat-v15-latin-300.woff2") format("woff2"),
+    url("/fonts/montserrat-v15-latin-300.woff") format("woff"),
+    url("/fonts/montserrat-v15-latin-300.ttf") format("truetype"),
+    url("/fonts/montserrat-v15-latin-300.svg#Montserrat") format("svg");
+}
+
+@font-face {
+  font-family: "Montserrat";
+  font-style: normal;
+  font-weight: 500;
+  font-display: swap;
+  src: url("/fonts/montserrat-v15-latin-500.eot");
+  src: local(""),
+    url("/fonts/montserrat-v15-latin-500.eot?#iefix")
+      format("embedded-opentype"),
+    url("/fonts/montserrat-v15-latin-500.woff2") format("woff2"),
+    url("/fonts/montserrat-v15-latin-500.woff") format("woff"),
+    url("/fonts/montserrat-v15-latin-500.ttf") format("truetype"),
+    url("/fonts/montserrat-v15-latin-500.svg#Montserrat") format("svg");
+}
+
+@font-face {
+  font-family: "Montserrat";
+  font-style: normal;
+  font-weight: 600;
+  font-display: swap;
+  src: url("/fonts/montserrat-v15-latin-600.eot");
+  src: local(""),
```

The `local` function will search if the user device already has the font family and then loaded instead of fetching the next source. But here it's receiving an empty string so it won't work.
 
```suggestion
  src: local("Montserrat"),
```